src/screens/SearchResult.js

```diff
@@ -43,7 +43,11 @@ const SearchResult = () => {
     }
   }, [loading]);
 
-  return loading ? <CircularProgress className="searchResult-loadingScreen" style={{width:"91%",height:"5em"}}/> : displayCatalogs();
+  return loading ? (
+    <CircularProgress className="searchResult-loadingScreen" />
```

Comment:
Removed inline CSS and gave width and other properties in form of %. Had to give height as "em" else height was not working with % values